docs/examples/rx/10_subscription_response/index.js

```diff
@@ -0,0 +1,24 @@
+const { zip } = require('rxjs');
+const { ApiRx } = require('@cennznet/api');
+const assert = require('assert').strict;
+
+async function main () {
+    // Initialise the provider to connect to the Azalea node
+    const provider = 'wss://cennznet.unfrastructure.io/public/ws';
+    // Create the API and wait until ready
+    const api = await ApiRx.create({ provider }).toPromise();
+
+    // We're using RxJs 'zip()' combination operator to get the emitted values
+    // of multiple observables as an array
+    // Pinned rpc-provider at 2.7.1 "@polkadot/rpc-provider": "2.7.1"
+    zip(
+        api.query.genericAsset.stakingAssetId(),
+        api.query.genericAsset.spendingAssetId()
```

Comment:
Can we stay focused on the actual query that failed Sylo?

Reply:
I didn't use that cause output can change... if an extrinsic is executed which updates the state.. the reason using these queries was cause they can't change..

Reply:
We can either try to separate the changeable part or query a specific entry in the history where we know the result.

Reply:
done